ev3dev/helper.py

```diff
@@ -146,6 +147,171 @@ class MediumMotor(ev3dev.auto.MediumMotor, MotorMixin):
     pass
 
 
+class MotorSet(object):
+    """
+    motor_specs is a dictionary such as
+    {
+        OUTPUT_A : LargeMotor,
+        OUTPUT_B : MediumMotor,
+    }
+    """
+
+    def __init__(self, motor_specs, desc=None):
```

Comment:
Each motor object can tell you what port it's on... why does this need to take a dictionary?

Reply:
The motor objects haven't been created yet though, we need to know what type of objects to make and which ports each of those objects is associated with.

Reply:
Sorry, I read through the rest of the file then forgot to come back and delete this. I've got it now.